src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -182,6 +187,39 @@ DynamicInstrumentationMethod CaptureOptionsDialog::GetDynamicInstrumentationMeth
   ORBIT_UNREACHABLE();
 }
 
+void CaptureOptionsDialog::SetWineSyscallHandlingMethod(
+    orbit_client_data::WineSyscallHandlingMethod method) {
+  switch (method) {
+    case WineSyscallHandlingMethod::kNoSpecialHandling:
+      ui_->wineNoneRadioButton->setChecked(true);
+      break;
+    case WineSyscallHandlingMethod::kStopUnwinding:
+      ui_->wineStopRadioButton->setChecked(true);
+      break;
+    case WineSyscallHandlingMethod::kRecordUserStack:
+      ui_->wineRecordRadioButton->setChecked(true);
+      break;
+    default:
```

Comment:
Curiosity: `default` is needed also with `enum class`es?

Reply:
Good catch, since its a class now, it is not needed. In particular, there will be an error when I miss a case.